src/types/PresignedPostRequest.ts

```diff
@@ -1,10 +1,11 @@
 import { ajv } from '../ajv';
+import { uuidSchema } from './Uuid';
 import type { JSONSchemaType } from 'ajv';
 import type { PresignedPost } from '@aws-sdk/s3-presigned-post';
+import type { Uuid } from './Uuid';
 
 export interface PresignedPostRequest {
-	fileType: string;
-	fileSize: number;
+	fileUuid: Uuid;
```

Comment:
The removal of two fields and addition of one required field to the Pre-signed post endpoint is a breaking change. We should increment the major version of the software, perhaps to 1.0.0 if we haven't already released it.

Reply:
We aren't releasing major versions at this point, but yes this will be a version bump.

I wouldn't want to couple this particular change with a decision to exit the 0.x semver stage.